core/drivers/sub.mk

```diff
@@ -47,5 +47,6 @@ srcs-$(CFG_ZYNQMP_HUK) += zynqmp_huk.c
 subdirs-y += crypto
 subdirs-$(CFG_BNXT_FW) += bnxt
 subdirs-$(CFG_DRIVERS_CLK) += clk
+subdirs-$(CFG_DRIVERS_RSTCTRL) += rstctrl
```

Comment:
Shouldn't `CFG_DRIVERS_RSTCTRL` be documented in `config.mk` ?

Reply:
yes it should. i'll add. thanks.